.changeset/cold-crabs-arrive.md

```diff
@@ -0,0 +1,6 @@
+---
+'@astrojs/markdown-remark': minor
+'astro': minor
+---
+
+Enable the "defaultColor" option from shiki
```

Comment:
Just noting that there will need to be more here, if this is something the user can now configure! I will wait for the Docs PR to see what all is involved before making any suggestions here!